### PR TITLE
Update chain-registry to v15.1.2

### PR DESCRIPTION
**Pull request by niccoloraspa**
## What is the purpose of the change

Update chain-registry to latest minor release `v15.1.2`

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Comment by niccoloraspa** [comment]
@JeremyParish69 could you please review if the format is correctly?
This should automatically be parsed by the chain-registry repo correct?

**Review by p0mvn — APPROVED** [review]

File: chain.schema.json [hunk 1]
```diff
@@ -83,28 +84,8 @@
...
-          "type": "tendermint",
-          "version": "0.34"
-        },
-        "cosmwasm_version": "0.30",
-        "cosmwasm_enabled": true,
-        "ibc_go_version": "4.3.0",
-        "ics_enabled": [
-          "ics20-1"
-        ],
-        "binaries": {
```
Comment (JeremyParish69) [inline chain.schema.json:104]: can we please update the binaries here is the version object and keep all the other properties (sdk version, cw version, consensus, ibc version, etc.)?

Comment (czarcas7ic) [reply chain.schema.json:104]: pinging @niccoloraspa for vis

**Comment by ValarDragon** [comment]
Closing for staleness, I think we remake the PR

**Pull request closed without merging**